src/index.ts

```diff
@@ -0,0 +1,17 @@
+import express, { Express } from 'express';
+
+import router from './routes';
+import swagger from './swagger';
+
+const app: Express = express();
+
+const hostname: string = '127.0.0.1';
+const port: number = 3000;
```

Comment:
Such numbers better declare like constants in separate module like `constants.ts` and import like:

```suggestion
import {HOST, PORT} from 'constants.ts';

const hostname: string = HOST;
const port: number = PORT;
```

Reply:
thanks for the feedback, I'll fix it now